frontend/src/components/features/chat/interactive-chat-box.tsx

```diff
@@ -38,6 +39,7 @@ export function InteractiveChatBox({
   const handleSubmit = (message: string) => {
     onSubmit(message, images);
     setImages([]);
+    onChange?.("");
```

Comment:
I don't think adding it here is a good idea. If I've added an image accidentally and removed it, all my text data is gone too